pkg/cloudprovider/provider/aws/provider.go

```diff
@@ -300,6 +302,19 @@ func (p *provider) getConfig(s v1alpha1.ProviderSpec) (*Config, *providerconfig.
 	if err != nil {
 		return nil, nil, nil, err
 	}
+	if c.DiskType == ec2.VolumeTypeIo1 {
+		if c.DiskIops == nil {
```

Comment:
You need to check for `rawConfig.DiskIOPS` here, `c.DiskIOps` is always nil at this point. The same for the assignment on l309

Reply:
oh: yes, how silly :) pushed a fix now, Let me know if you want me to squash the commits.

Reply:
Doesn't matter, it gets all squashed together on merge